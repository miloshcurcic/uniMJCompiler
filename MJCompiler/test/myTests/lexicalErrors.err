ERROR 20:23:42,772 - Line 5 Column 5: Lexical analysis error for text "$#@#"!
ERROR 20:23:42,778 - Line 10 Column 6: Lexical analysis error for text "@#$"!
ERROR 20:23:42,779 - Line 11 Column 6: Lexical analysis error for text "@$@"!
