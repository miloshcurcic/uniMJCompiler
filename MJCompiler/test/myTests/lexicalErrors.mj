//Test 302
// all errors
program recovery

int $#@# something;

{
	void main()
	{
	    @#$
	    @$@
    }
}

