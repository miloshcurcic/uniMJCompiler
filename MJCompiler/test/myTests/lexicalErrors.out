INFO  20:20:11,818 - ===================================
INFO  20:20:11,821 - Compiling source file: H:\Fakultet\Semestar 7\PP\MJCompiler\test\test304.mj
INFO  20:20:11,821 - ===================================
ERROR 20:20:11,884 - Line 5 Column 5: Lexical analysis error for text "$#@#"!
ERROR 20:20:11,886 - Line 10 Column 6: Lexical analysis error for text "@#$"!
ERROR 20:20:11,887 - Line 11 Column 6: Lexical analysis error for text "@$@"!
INFO  20:20:11,888 - ===================================
INFO  20:20:11,888 - One or more lexical or syntax errors detected.
INFO  20:20:11,888 - ===================================
